test/simperium/default_storage_test.js

```diff
@@ -0,0 +1,35 @@
+import assert from 'assert'
+import Bucket from '../../src/simperium/bucket'
+import defaultStore from '../../src/simperium/storage/default'
+
+describe( 'default store', function() {
+	var bucket;
+
+	beforeEach( function() {
+		bucket = new Bucket( 'things', defaultStore );
+	} )
+
+	it( 'should store object update', function( done ) {
+		var id = 'thing',
+			data = {one: 'two'};
+
+		bucket.update( id, data, function() {
+			bucket.get( id, function( err, object ) {
+				assert.deepEqual( object, { data, id } );
+				done();
+			} );
+		} );
+	} );
+
+	it( 'should update with options', function( done ) {
+		var id = 'thing',
+			data = {one: 'two'}
+
+		bucket.update( id, data, {}, function() {
```

Comment:
I'm confused here because the function declaration shows `( id, object, isIndexing, callback )` where I presume `isIndexing` should be a boolean value, but this line calls `( id, object, {}, callback )`.

Is that intentional?